content/events/2018/tech-day-02-17.md

```diff
@@ -0,0 +1,17 @@
+---
+title: February Tech Day 2
+date: 2018-02-17
+---
+
+{{< location 25 >}}
+
+The day will run from 10:00am until 5:00pm, however feel free to come and go as you please.
+
+## Food
+Food on campus is scarce on weekends, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however. There's a [map of local food places](https://goo.gl/yYlfs5) to help you find nearby food.
+
+## Electronics
+As expected, you'll need to bring your robot to work on, in whatever state it may be. We also recommend bringing at least 1 laptop with you to work on. Laptops will be provided for you to use.
+
+## Supervision
```

Comment:
This snippet is going to be the same across all our events. Maybe pull out somehow to deduplicate? (As-is is fine for now, though we should keep an eye on whether we're creating more duplicates)

Reply:
I'll extract it into a shortcode, so we can specifically include it when needed